### ISSUE TITLE
 Type not found in haxeui-core/git/haxe/ui/Toolkit.hx

### ISSUE THREAD

**Issue by gshields** [haxeui/haxeui-core#216]
## Possible Solution

haxelib set haxeui-core 0.0.3
## Steps to Reproduce (for bugs)

E:\Documents\work_src\openLR>openfl build cpp
E:/pasco-software/p-tools/HaxeToolkit/haxe/lib/haxeui-core/git/haxe/ui/Toolkit.hx:3: characters 7-37 : Type not found : haxe.ui.backend.ToolkitOptions

## Your Environment

windows 10
E:\Documents\work_src\openLR>haxelib list
actuate: [1.8.7]
box2d: [1.2.3]
haxeui-core: 0.0.3 [git]
haxeui-openfl: [0.0.2]
hscript: [2.1.1]
hxcpp: 3.4.185 [3.4.188]
layout: [1.2.1]
lime-samples: [6.2.0]
lime: [6.2.0]
nme: [6.0.58]
openfl-samples: [6.0.0]
openfl: [7.1.2]
refactor: [4.0.0]

E:\Documents\work_src\openLR>haxe -version
3.4.7

**Comment by ianharrigan** [comment]
You need the git version of haxeui-openfl too.

**Comment by Dunkhan** [comment]
I have the same issue. I am definitely using the git version of haxeui-openfl

**Comment by Dunkhan** [comment]
Yeah that file is there as expected. I just did a whole system reinstall so I most likely failed to link something correctly. I just noticed that there is an option marked "Add source path" in haxedevelop so I tried that on this folder (despite having used the command "haxelib dev haxeui-openfl blah" already) and this resolved the issue.
Unfortunately this was then replaced with a new issue, the ToolkitOptions.hx file now has an error "Type not found : openfl.display.Stage" in its only import statement. Any tips on how to resolve this now?
